test/verify/check-system-info

```diff
@@ -873,54 +873,38 @@ password=foobar
 
     @skipImage("crypto-policies not avaiable", "fedora-coreos", "debian-stable", "debian-testing", "ubuntu-2004", "ubuntu-2204", "ubuntu-stable", "arch")
     def testCryptoPolicies(self):
+        m = self.machine
+        b = self.browser
+        self.allow_journal_messages()
+
         def shown_profile_text(profile):
-            return profile[0] + profile[1:].lower()
+            return profile if profile == "FIPS" else profile[0] + profile[1:].lower()
 
-        def change_profile(profile, new_profile, reboot=False):
+        def change_profile(profile, new_profile):
             b.click("#crypto-policy-button")
             b.wait_in_text(".pf-c-menu__item.pf-m-selected", shown_profile_text(profile))
             profile_button_name = shown_profile_text(new_profile)
             b.click(f".pf-c-menu__item-main .pf-c-menu__item-text:contains('{profile_button_name}')")
-            if reboot:
-                b.click("#crypto-policy-save-reboot")
+            b.click("#crypto-policy-save-reboot")
+            if new_profile == "FIPS":
+                # Initramfs re-generation takes a while
+                m.wait_reboot(timeout_sec=600)
```

Comment:
When disabling FIPS it certainly also has to regenerate the initrd?

Reply:
It doesn't seem to be the case, what I think is happening is that some "FIPS" module has to be loaded in initramfs when enabled and a boot flag set and disabling only switches the boot flag? I'd have to do some more digging to figure that out.